src/integrations/bluetooth-low-energy/bluetooth-low-energy.service.ts

```diff
@@ -106,18 +106,24 @@ export class BluetoothLowEnergyService extends KalmanFilterable(Object, 0.8, 15)
    * @param event - Event with new distance data
    */
   handleNewDistance(event: NewDistanceEvent): void {
-    const sensorId = makeId(`ble ${event.tagId}`);
+    const sensorId = makeId(`ble ${event.tagName}`);
```

Comment:
This will change the IDs for all tags, not just the ones using the name. This would mean that in Home Assistant new devices/entities would appear for everybody already using the software. Plus, it could potentially break some scenarios in which multiple peripherals with the same name are being tracked by their IDs.